formats/cbor/commonMain/src/kotlinx/serialization/cbor/Cbor.kt

```diff
@@ -32,13 +32,14 @@ import kotlinx.serialization.modules.*
 public sealed class Cbor(
     internal val encodeDefaults: Boolean,
     internal val ignoreUnknownKeys: Boolean,
+    internal val allowDuplicateKeys: Boolean,
     override val serializersModule: SerializersModule
 ) : BinaryFormat {
 
     /**
      * The default instance of [Cbor]
      */
-    public companion object Default : Cbor(false, false, EmptySerializersModule())
```

Comment:
Is there a convention in this library that the defaults should be false, or is the introduction of a `true`-defaulting setting OK? (Wasn't sure whether to go with allow or forbid in the name.)

Reply:
Leaning towards switching this to `forbidDuplicateKeys`, actually.